src/sentry/uptime/detectors/tasks.py

```diff
@@ -245,14 +245,7 @@ def monitor_url_for_project(project: Project, url: str) -> ProjectUptimeSubscrip
     it. Also deletes any other auto-detected monitors since this one should replace them.
     """
     for monitored_subscription in get_auto_monitored_subscriptions_for_project(project):
-        delete_uptime_subscriptions_for_project(
-            project,
-            monitored_subscription.uptime_subscription,
-            modes=[
-                ProjectUptimeSubscriptionMode.AUTO_DETECTED_ONBOARDING,
-                ProjectUptimeSubscriptionMode.AUTO_DETECTED_ACTIVE,
-            ],
-        )
+        delete_project_uptime_subscription(monitored_subscription)
```

Comment:
Right now, this code is never called because we only ever detect a single url for a project. But this is the correct behaviour if we ever change that.